src/soc/ibm/power9/timer.c

```diff
@@ -0,0 +1,7 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+
+#include <delay.h>
+
+void init_timer(void)
+{
```

Comment:
Timer is already implemented in the src/arch/ppc, see my commit. If there is something related to init_timer there, then move it here. If not, leave a comment that no need to do anything here and arch timer doesn't need any initialization

Reply:
Added a comment